internal/controller/streams.go

```diff
@@ -380,15 +442,18 @@ func ConnectStreams(ctx context.Context, logger *zap.SugaredLogger, envMap Envir
 
 			resourceDone := make(chan struct{})
 			logDone := make(chan struct{})
+			falcoDone := make(chan struct{})
 			var ciliumDone chan struct{}
 			sm.bufferedGrpcSyncer.done = logDone
 
 			go manageStream(logger, connectAndStreamResources, sm, resourceDone)
 			go manageStream(logger, connectAndStreamLogs, sm, logDone)
+			go manageStream(logger, connectAndStreamFalcoNetworkFlows, sm, falcoDone)
 			// Only start network flows stream if not disabled
-			if !sm.streamClient.disableNetworkFlows {
+			if !sm.streamClient.disableNetworkFlowsCilium {
 				ciliumDone = make(chan struct{})
 				go manageStream(logger, connectAndStreamCiliumNetworkFlows, sm, ciliumDone)
+
```

Comment:
please add the falcoDone channel to the select at line 464 for reconnect logic

Reply:
Good catch